### PR TITLE
fix(comment-syntax): check list item alignment

### PR DESCRIPTION
**Pull request by d3lm**
This PR fixes an issue with our `comment-syntax` rule where the alignment of list items was not checked / taken into account and each line was expected to be capitalized because it was seen as a new paragraph.

Fixes WC-443

### CODE REVIEW

File: src/rules/comment-syntax.ts [hunk 1]
```diff
@@ -220,7 +223,7 @@ export default createRule<Options, MessageIds>({
                 continue;
               }
 
-              if (isCurrentLineJSDoc) {
+              if (isCurrentLineJSDoc && !insideCodeBlock) {
```
Comment (d3lm) [inline src/rules/comment-syntax.ts:226]: An unrelated fix. It didn't allow to use `@` inside a code block, e.g.
```
/**
 *```
 * @ foo
 *```
 */
```

**Review by Nemikolh — APPROVED** [review]
Amazing!! :tada: You did that in record time :clap:

Let's GO :sunglasses:

**Comment by d3lm** [comment]
That review was also in record time 🔥 🙌
